tests/integration_tests.py

```diff
@@ -22,28 +22,28 @@ def mock_sagemaker_session_(*args, **kwargs):
 
 
 class TestPackageActions:
-    endpoint_name = 'TestEndpoint'
-    test_instance_type = 'ml.m5.large'
-    test_role = 'arn:aws:iam::311638508164:role/AmazonSageMaker-ExecutionRole'
+    endpoint_name = "TestEndpoint"
+    test_instance_type = "ml.m5.large"
+    test_role = "arn:aws:iam::311638508164:role/AmazonSageMaker-ExecutionRole"
```

Comment:
Guys, let's not expose AWS account IDs 🙏

Reply:
Get rid of them everywhere, please 🙏

Reply:
Not resolved. Resolve or put into a separate task